R/plotly.R

```diff
@@ -336,6 +338,8 @@ plotly_build <- function(l = last_plot()) {
       ) %||%
       "plot from api" 
   }
+  # place kwargs back into x
+  x[get_kwargs()] = kwargs[get_kwargs()]
   # traces shouldn't have any names
```

Comment:
This can cause empty list elements, which can cause problems. For example,
```
x <- list(a = 1)
y <- list(a = 1)
x["b"] <- y["b"]
x

# $a
# [1] 1
#
# $b
# NULL
```